src/controller/CommissionerDiscoveryController.h

```diff
@@ -235,6 +235,19 @@ class DLL_EXPORT AppInstallationService
      */
     virtual bool LookupTargetContentApp(uint16_t vendorId, uint16_t productId) = 0;
 
+    /**
+     * @brief
+     *   Called to check app's installation status.
+     *
+     * This will be called by the main chip thread so any blocking work should be moved to a separate thread.
```

Comment:
since this method is synchronous, is it reasonable to expect blocking work to be moved to a separate thread?

Reply:
Updated.